include/gbinder_reader.h

```diff
@@ -169,6 +169,11 @@ gboolean
 gbinder_reader_skip_string16(
     GBinderReader* reader);
 
+const void*
+gbinder_reader_read_byte_array(
+    GBinderReader *reader,
```

Comment:
The rest of the code is using `int* x` style (as opposed to `int *x`) but that's a minor aesthetical thing. Feel free to ignore it, I can clean those up that myself in a separate commit.